dokka-templates/base.ftl

```diff
@@ -8,10 +8,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" charset="UTF-8">
     <@page_metadata.display/>
     <@template_cmd name="pathToRoot"><script>var pathToRoot = "${pathToRoot}";</script></@template_cmd>
+    <link rel="preload" href="/_assets/dokka-template.css" as="style">
+    <link rel="preload" href="${pathToRoot!""}/navigation.html" as="document"/>
     <script>document.documentElement.classList.replace("no-js","js");</script>
     <#-- This script doesn't need to be there but it is nice to have
     since app in dark mode doesn't 'blink' (class is added before it is rendered) -->
-    <script>const storage = localStorage.getItem("dokka-dark-mode")
+    <script async>const storage = localStorage.getItem("dokka-dark-mode")
```

Comment:
Doesn't making the script async interfere with the intention to fix the "blink" when the theme is dark? (see the comment above)